src/hooks/useTrackSectionStarted.ts

```diff
@@ -0,0 +1,73 @@
+import { PupilExperienceLessonActivityValueType } from "@/browser-lib/avo/Avo";
```

Comment:
Hmmm this looks like 3 hooks disguised as one. Can we split them into separate files pls.  But actually I think we don't need trackSectionStarted at all anymore. Rather just call those methods directly from the various views.